### PR TITLE
[FHB-50] Referral search design updates

### PR DESCRIPTION
**Pull request by Top-Cat**

### CODE REVIEW

File: src/ui/connect-ui/src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/LocalOfferResults.cshtml [hunk 1]
```diff
@@ -427,4 +432,12 @@ else
     </div>
     </div>
 }
-</form>
\ No newline at end of file
+</form>
+
+@functions {
+    // todo: move to sd shared
```
Comment (stevesatdfe) [inline src/ui/connect-ui/src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/LocalOfferResults.cshtml:438]: Comment need to be removed?

File: src/ui/connect-ui/src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/LocalOfferResults.cshtml.cs [hunk 2]
```diff
@@ -361,14 +362,21 @@ public static string GetDeliveryMethodsAsString(ICollection<ServiceDeliveryDto>
     {
         return serviceDeliveries.Count == 0
             ? string.Empty
-            : string.Join(",", serviceDeliveries.Select(serviceDelivery => serviceDelivery.Name.AsString(EnumFormat.Description)));
+            : string.Join("\n", serviceDeliveries.Select(serviceDelivery => serviceDelivery.Name.AsString(EnumFormat.Description)));
```
Comment (stevesatdfe) [inline src/ui/connect-ui/src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/LocalOfferResults.cshtml.cs:365]: Use Environment.NewLine as this handles platform variances.

File: src/ui/connect-ui/src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/LocalOfferResults.cshtml.cs [hunk 3]
```diff
@@ -361,14 +362,21 @@ public static string GetDeliveryMethodsAsString(ICollection<ServiceDeliveryDto>
...
+    }
+
+    public static string GetAddressAsString(ServiceDto serviceDto)
+    {
+        return serviceDto.Locations.Count == 1
+            ? string.Join("\n", serviceDto.Locations.First().GetAddress())
```
Comment (stevesatdfe) [inline src/ui/connect-ui/src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/LocalOfferResults.cshtml.cs:371]: Use Environment.NewLine as this handles platform variances.

**Review by stevesatdfe — CHANGES_REQUESTED** [review]

File: src/ui/connect-ui/src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/LocalOfferResults.cshtml (hunk 1 above)
Comment (Top-Cat) [reply src/ui/connect-ui/src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/LocalOfferResults.cshtml:438]: It's not in sd shared yet

**Review by stevesatdfe — APPROVED** [review]

File: src/ui/connect-ui/src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/LocalOfferResults.cshtml [hunk 4]
```diff
@@ -427,4 +432,12 @@ else
...
+    private static string AgeToString(int age)
+    {
+        return age == 127 ? "25+" : age.ToString();
+    }
```
Comment (AaronJY) [inline src/ui/connect-ui/src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/LocalOfferResults.cshtml:442]: Question: where does 127 come from, and why do we treat it as 25+?

**Review by Zac-AND-Digital — DISMISSED** [review]

File: src/ui/connect-ui/src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/LocalOfferResults.cshtml (hunk 4 above)
Comment (Top-Cat) [reply src/ui/connect-ui/src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/LocalOfferResults.cshtml:442]: This is a copy of another function. So I don't really have answers to that other than we do.

**Review by Zac-AND-Digital — DISMISSED** [review]

**Review by AaronJY — APPROVED** [review]

**Pull request merged**